src/VaultBridgeToken.sol

```diff
@@ -912,14 +912,14 @@ abstract contract VaultBridgeToken is
     }
 
     /// @notice Transfers yield generated by the yield vault to the yield recipient in the form of vbToken.
-    /// @notice The reserve will be rebalanced after collecting yield.
+    /// @notice Does not rebalance the reserve after collecting yield to allow usage while the contract is paused.
     /// @notice This function can be called by the yield collector only.
-    function collectYield() external whenNotPaused onlyRole(YIELD_COLLECTOR_ROLE) nonReentrant {
+    function collectYield() external onlyRole(YIELD_COLLECTOR_ROLE) nonReentrant {
         _collectYield(true);
     }
 
     /// @notice Transfers yield generated by the yield vault to the yield recipient in the form of vbToken.
-    /// @dev The reserve will be rebalanced after collecting yield.
+    /// @dev Does not rebalance the reserve after collecting yield to allow usage while the contract is paused.
```

Comment:
@ZeroEkkusu doublecheck

Now `_rebalanceReserve()` is removed from `_collectYield()`, but the suggestion of L-04 was to remove it from `burn()`

Reply:
Good catch. I thought it had been removed.

Reply:
[4abf293](https://github.com/agglayer/vault-bridge/pull/21/commits/4abf293921cbd833a93b8bc0857a28235fc6bfac)